### ISSUE TITLE
Compatibility with Plots.jl

### ISSUE THREAD

**Issue by remoore** [davidavdav/ROCAnalysis.jl#6]
Is this package compatible with Plots.jl. I've tried running the following code (based on the README):

```
using ROCAnalysis
tar = 2 + 2randn(1000)
non = -2 + 2randn(100000)
r = roc(tar, non)
using Plots
plot(r)
```

This results in an error saying that there is no user recipe defined for `ROCAnalysis.Roc{Float64}`.

**Comment by davidavdav** [comment]
No, this is not yet using Plots. Is uses something else for plotting, Winston.  I am generally not happy with the plotting mess in Julia, but plots.jl looks promising.

**Comment by davidavdav** [comment]
I am now running into the same issue.  Did you happen to have looked into Plots?

**Comment by davidavdav** [comment]
Ah—I see that I already have some rudimentary support for `Plots`.  The plots still are quite ugly, because, well, not R, but I was able to make a basic DET plot.  

I should update the README accordingly.
